### PR TITLE
CHANGELOG for the 2022.06 release

### PR DESCRIPTION
**Pull request by tobiasd**
This PR shall be merged as the last one for the 2022.06 release

### CODE REVIEW

**Review by MrPetovan — APPROVED** [review]

**Review by MrPetovan — APPROVED** [review]

**Review by MrPetovan — APPROVED** [review]

**Review by MrPetovan — APPROVED** [review]

**Review by MrPetovan — APPROVED** [review]

**Review by MrPetovan — APPROVED** [review]

**Review by MrPetovan — APPROVED** [review]

File: CHANGELOG [hunk 1]
```diff
@@ -1,11 +1,63 @@
-Version 2022.05 (unreleased)
+Version 2022.06 (unreleased)
```
Comment (nupplaphil) [inline CHANGELOG:1]:
```suggestion
Version 2022.06
```
:-)

Comment (tobiasd) [reply CHANGELOG:1]: I would, but AFAIK there is still this very nasty bug discussed in the support forum about missing interactivity with postings.

Comment (annando) [reply CHANGELOG:1]: I digged through the code and fixed every possible or impossible source. The only thing left now is to completely overhaul the whole permission stuff. And this is something that I wouldn't like to do in the RC phase.

Concerning that issue: AFAIK very few people reported it. And even fewer reported that it still only partially works. So my hope is that it won't affect too many users.

Comment (tobiasd) [reply CHANGELOG:1]: Ok, after inquireing the status of the problem it seems to be solved. Though there may be lingering some problem with Diaspora* or may not... anyway I'll to the release tomorrow morning CEST. Testinstallation on Debian Stable is working fine :)

Comment (MrPetovan) [reply CHANGELOG:1]: Thank you for your work!

**Pull request merged**